core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -30,6 +30,10 @@ public String getAmbassadorContainerImage() {
         return (String) properties.getOrDefault("ambassador.container.image", "richnorth/ambassador:latest");
     }
 
+    public String getSocatContainerImage() {
+        return (String) properties.getOrDefault("socat.container.image", "alpine/socat:latest");
```

Comment:
I had to use `:latest` :(
https://hub.docker.com/r/alpine/socat/tags/

Even tho there is a tag, it seems to be old and didn't work for me. But I assume it's fine to use `latest` here because `socat` itself is supposed to be backward compatible and shouldn't break over the time

Reply:
once we get our name on Docker Hub - sure, for now I would stick with Alpine's